Modix.Data/Repositories/MessageRepository.cs

```diff
@@ -141,5 +144,16 @@ private IQueryable<MessageEntity> GetGuildUserMessages(ulong guildId, ulong user
             return ModixContext.Messages.AsNoTracking()
                 .Where(x => x.GuildId == guildId && x.AuthorId == userId && x.Timestamp >= earliestDateTime);
         }
+
+        public async Task<MessageEntity> GetMessage(ulong messageId)
+        {
+            return await ModixContext.Messages.FindAsync(messageId);
+        }
+
+        public void UpdateStarboardColumn(MessageEntity message)
```

Comment:
It might be nice to implement the `Action<XMutationData>` pattern we use in some of the other repositories here. We try not to expose `XEntity` outside of Modix.Data.dll.

Reply:
I'll check it out.